core/src/main/java/org/ehcache/Ehcache.java

```diff
@@ -1075,28 +1074,12 @@ public CacheStatistics getStatistics() {
 
   void create() {
     statusTransitioner.checkMaintenance();
-    try {
-      if (store instanceof Persistable) {
-        logger.info("Creating Cache storage.");
-        ((Persistable) store).create();
-        logger.info("Cache storage successfully created.");
-      }
-    } catch (Exception e) {
-      throw new RuntimeException("Couldn't create Cache storage ", e);
-    }
+    // TODO figure out persistence and user managed caches
```

Comment:
If I recall it right , this is just the partial PR which talks about API stuff. There will be further PRs for the same issue